src/server.c

```diff
@@ -1976,6 +1976,8 @@ void initServerConfig(void) {
     server.bindaddr_count = CONFIG_DEFAULT_BINDADDR_COUNT;
     for (j = 0; j < CONFIG_DEFAULT_BINDADDR_COUNT; j++) server.bindaddr[j] = zstrdup(default_bindaddr[j]);
     memset(server.listeners, 0x00, sizeof(server.listeners));
+    server.host_machine_ip = 0;
+    server.host_machine_netmask = 0;
```

Comment:
I see that you added some of logic to retrieve it, but could not find what this is used for.